code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1185,6 +1190,89 @@
 		M.confused = min(M.confused + 2, 5)
 	..()
 
+/datum/reagent/carbon_monoxide
+	name = "Carbon Monoxide"
+	description = "A highly dangerous gas for sapients."
+	reagent_state = GAS
+	metabolization_rate = 0.1 * REAGENTS_METABOLISM
+	color = "#96898c"
+	var/accumilation
+
+/datum/reagent/carbon_monoxide/on_mob_life(mob/living/carbon/victim)
+	if(holder.has_reagent(/datum/reagent/oxygen))
+		holder.remove_reagent(/datum/reagent/carbon_monoxide, 2*REM)
+		accumilation = accumilation/4
```

Comment:
This is double dipped w/ oxygen reagent on_mob_life and I think it should be handled in one or the other